py-polars/polars/dataframe/frame.py

```diff
@@ -607,6 +610,55 @@ def plot(self) -> hvPlotTabularPolars:
         hvplot.post_patch()
         return hvplot.plotting.core.hvPlotTabularPolars(self)
 
+    @property
+    def style(self) -> GT:
```

Comment:
could you also add `@unstable` please, along with a note about it in the docstring?

this should probably have happened in the `plot` one, but just got missed

if you check some other 3rd party methods, like `to_jax`, you should be able to copy from there

Reply:
Thanks for the pointer to `to_jax()`! I added unstable and put the same warning inside the `.style` docstring.